node/tessel-export.js

```diff
@@ -952,6 +957,303 @@ Tessel.LED.prototype.read = function(callback) {
   });
 };
 
+Tessel.Wifi = function() {
+  var state = {
+    settings: {},
+    connected: false
+  };
+
+  Object.defineProperties(this, {
+    isConnected: {
+      get: () => state.connected
+    },
+    connected: {
+      set: (value) => {
+        state.connected = value;
+      }
+    },
+    settings: {
+      get: () => state.settings,
```

Comment:
This will actually expose the reference, which makes it vulnerable to unprivileged access (modification, etc).